docs/UserGuide.adoc

```diff
@@ -135,9 +135,23 @@ Expected output:
 
 * Adds the expense spent on Laksa back to THRIFT.
 
+=== List expenses for the month: `list`
+
+Format: `list`
+
+Examples:
+
+*  `list`
+*  `list m/Jan t/Food`
+
+Expected output:
+
+* Returns a list of all expenses for the current month and remaining budget left.
```

Comment:
I think remaining budget will be shown on the Ui at all times, so maybe do not have to state it here which might lead to confusion.